src/simulate/send_to_udp/HttpHandler.py

```diff
@@ -1,34 +1,37 @@
 import logging
 import requests
 import utils
+import time
 
 
 class HttpHandler:
-    def __init__(self, config_data):
+    def __init__(self, config_data, session):
         self.config_data = config_data
+        self.session = session
 
     def make_api_call(self, data):
         response = None
         try:
-            http = self.config_data[utils.PROPS_ENDPOINT]
+            endpoint = self.config_data[utils.PROPS_ENDPOINT]
         except KeyError:
             logging.error('configuration yaml is missing the\"' + utils.PROPS_ENDPOINT + '\"key')
```

Comment:
Do you want spaces before/after the quotes?

Reply:
There's no question.  It needs spaces.

Reply:
Done